g6/auditInformationProvided.yml

```diff
@@ -3,16 +3,14 @@ filters: 'Consumers are aware of the audit information that will be provided to
 hint: 'Do you provide audit information to consumers? Choose 1.'
 dependsOnLots: 'SaaS, PaaS, IaaS'
 assuranceApproach: 2answers-type1
-fields:
+type: radios
+options:
     -
-        type: radio
         label: None
```

Comment:
Sorry, Sorry - i remember now.  FilterLabel indicates if users can filter on these options.  Sorry, as you were.